pkgs/development/compilers/elm/packages/generate-node-packages.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p nodePackages.node2nix
+
+set -eu -o pipefail
+
+rm -f node-env.nix
+node2nix --nodejs-10 -i node-packages.json -o node-packages.nix -c node-composition.nix
```

Comment:
similar to https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/node-packages/generate.sh